File: apprefurbished/src/main/java/com/androidcollider/easyfin/managers/format/number/NumberFormatModule.java
package com.androidcollider.easyfin.managers.format.number;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class NumberFormatModule {

    @Provides
    NumberFormatManager provideDoubleFormatManager() {
        return new NumberFormatManager();
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/format/number/NumberFormatModule.kt
package com.androidcollider.easyfin.common.managers.format.number

import dagger.Module
import dagger.Provides

/**
 * @author Ihor Bilous
 */
@Module
class NumberFormatModule {
    @Provides
    fun provideDoubleFormatManager(): NumberFormatManager {
        return NumberFormatManager()
    }
}